StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Utils/LinkURLGenerator.swift

```diff
@@ -82,7 +89,14 @@ class LinkURLGenerator {
         let paymentObjectType: LinkURLParams.PaymentObjectMode = elementsSession.linkPassthroughModeEnabled ? .card_payment_method : .link_payment_method
 
         let intentMode: LinkURLParams.IntentMode = intent.isPaymentIntent ? .payment : .setup
-
+        var cardBrandChoiceInfo: LinkURLParams.CardBrandChoiceInfo?
+        
+        if let cardBrandChoice = elementsSession.cardBrandChoice {
+            cardBrandChoiceInfo = LinkURLParams.CardBrandChoiceInfo(isMerchantEligibleForCBC: cardBrandChoice.eligible, stripePreferredNetworks: (cardBrandChoice.allResponseFields["preferred_networks"] as? [String]) ?? [], supportedCobrandedNetworks: cardBrandChoice.allResponseFields["supported_cobranded_networks"] as? [String : Bool] ?? [:])
```

Comment:
Let's update `STPCardBrandChoice` to decode preferred networks and supported cobranded networks, and update it's test file with some tests.